index-en.md

```diff
@@ -382,8 +382,19 @@ css:
 	{% endfor %}
 	</ul>
 	<h2 id="other">Other documentation</h2>
-	<h3>WET-BOEW feature demos styled with Canada.ca theme</h3>
-	<p><a href="/gcweb-compiled-demos/index.html#wet-boew">WET-BOEW feature overview</a></p>
+	<div class="row mrgn-tp-md">
+		<div class="col-md-8">
+			<div class="row d-flex">
+				<div class="col-xs-3 col-md-2 d-flex align-self-center">
+					<img src="https://raw.github.com/wet-boew/wet-boew-attachments/master/Promo/WET_Logo.png" class="full-width" alt="">
+				</div>
+				<div class="col-xs-9 col-md-10">
+					<h3 class="mrgn-tp-md" id="wet-boew-demos">WET features styled with Canada.ca theme</h3>
+					<p><a href="/gcweb-compiled-demos/index.html#wet-boew">view list of demos</a></p>
+				</div>
+			</div>
+		</div>
+	</div>
```

Comment:
nice look

![image](https://user-images.githubusercontent.com/1676030/197669236-a94b8f81-e9e6-45a9-a375-79244757fc18.png)

It will be nice to associate the image along with the h3. May be with a `<section>` element?
And probably we can use the `stretched-link` too

Reply:
@duboisp Yes it would make sense to add a `<section>` element. The stretch link however would not include the image in the clickable area because columns are relatively positioned and stop the propagation of the stretch link.